app/src/main/java/com/worldsnas/starplayer/di/LocalMusicModule.kt

```diff
@@ -0,0 +1,19 @@
+package com.worldsnas.starplayer.di
+
+import android.app.Application
+import android.content.ContentResolver
+import android.content.Context
+import com.worldsnas.starplayer.App
+import dagger.Module
+import dagger.Provides
+
+@Module
+class LocalMusicModule {
```

Comment:
esme monasebi nist benazaram!

hamchenin pishnahad mikonam to kotlin baraye module ha az `object` estefade konin
va roye function ha `@Jvmstatic` bezanid.

Reply:
Lotfan esm pishnehad bedid